changelogs/fragments/5444-passwordstore-options.yml

```diff
@@ -0,0 +1,2 @@
+minor_changes:
+  - "passwordstore lookup plugin - allow options to be passed lookup options instead of being part of the term strings (https://github.com/ansible-collections/community.general/pull/5444)."
```

Comment:
I really disagree this should be put under `minor_changes` as this requires a code refactor. It should be considered a breaking change.